source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,131 @@ Hardware Checklist
    :local:
    :depth: 2
 
-Stub - detail Hardware Checklist Items
+Use the following checklist when planning the hardware configuration for a production, distributed MinIO deployment.
+
+Considerations
+--------------
+
+When selecting hardware for your MinIO implementation, take into account the following factors:
+
+- Expected amount of data in tebibytes to store at launch
+- Expected growth in size of data for at least the next two years
+- Number of objects by average object size
+- Average retention time of data in years
+- Number of sites to be deployed
+
+Hardware Requirements and Recommendations
+-----------------------------------------
+
+.. note:: 
+
+   For specific hardware suggestions, refer to https://min.io/product/reference-hardware.
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - | Sufficient CPU cores to acheive performance goals
+       | MinIO recommends Dual Intel® Xeon® Scalable Gold CPUs (minimum 8 cores per socket)
+
+   * - :octicon:`circle`
+     - | Sufficient RAM to acheive performance goals
+       | MinIO recommends at least 128 GB of RAM 
+
+   * - :octicon:`circle`
+     - Four nodes or servers
+
+   * - :octicon:`circle`
+     - | SATA/SAS drives for capacity and NVMe SSDs for high-performance
+       | MinIO recommends a minimum of 8 drives per server
+
+   * - :octicon:`circle`
+     - | 25GbE network for capacity 
+       | 100GbE Network interface cards for high performance
+
+.. important:: 
+
+   Object storage operations are primarily throughput bound. 
+   MinIO takes full advantage of the modern hardware improvements such as AVX-512 SIMD acceleration, 100GbE networking, and NVMe SSDs when available.
+
+Hardware Tests to Complete
+--------------------------
+
+MinIO Diagnostics
+~~~~~~~~~~~~~~~~~
+
+Run the following command.
+If you have access to :ref:`SUBNET <minio-docs-subnet>`, you can upload the results there.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support diag ALIAS
+
+Drive Tests
+~~~~~~~~~~~
+
+Run these tests outside of MinIO to verify capabilities of your hardware.
+Test each drive independently on all servers to ensure they are identical in performance.
+Record the results for later reference.
+
+#. Uncached write test
+   
```

Comment:
We should try to say what this is doing. at a glance, it writes about 80K files of random data using ODIRECT to avoid any drive caching. If possible we should try to provide some output as guidance on what the results mean - @eco-minio might be able to help here. There are specific meaningful sections in the output.

Also instructions on how to clean up afterwards. Not sure if we can use `of=/dev/null` ?

Reply:
Created tables for each command to list what each part of the command is doing. Also added what to expect for the output, without giving a specific example.